snuba/writer.py

```diff
@@ -39,10 +40,32 @@ def __row_to_column_list(self, columns, row):
 
     def write(self, rows: Iterable[WriterTableRow]):
         columns = self.__schema.get_columns()
-        self.__connection.execute_robust("INSERT INTO %(table)s (%(colnames)s) VALUES" % {
-            'colnames': ", ".join(col.escaped for col in columns),
-            'table': self.__schema.get_table_name(),
-        }, [self.__row_to_column_list(columns, row) for row in rows], types_check=False)
+        self.__connection.execute_robust(
+            "INSERT INTO %(table)s (%(colnames)s) VALUES"
+            % {
+                "colnames": ", ".join(col.escaped for col in columns),
+                "table": self.__schema.get_table_name(),
+            },
+            [self.__row_to_column_list(columns, row) for row in rows],
+            types_check=False,
+        )
+
+
+class ClickHouseError(Exception):
```

Comment:
I can unify this with the driver error message if anybody feels strongly about it.